src/Core/Enums/BitwardenProductType..cs

```diff
@@ -0,0 +1,11 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace Bit.Core.Enums;
+
+public enum BitwardenProductType : byte
+{
+    [Display(Name = "PasswordManager")]
+    PasswordManager = 0,
+    [Display(Name = "SecretManager")]
+    SecretManager = 1,
```

Comment:
I'm thinking this should be `SecretsManager` for the enum value and display name - would anyone from @bitwarden/team-secrets-manager-dev care to weigh in?

Reply:
@vincentsalucci I would agree with recommending `SecretsManager` since that's the naming convention we use for code owner files/folders as well.

Reply:
Thank you @Thomas-Avery !!